### ISSUE TITLE
System.Reflection.TypeExtensions test fails with package upgrade to rc3-23915

### ISSUE THREAD

**Issue by Priya91** [dotnet/corefx#6932]
```
System.Reflection.Tests.PropertyInfoGetAccessor1.NegTest1 [FAIL]
               Assert.Throws() Failure
               Expected: typeof(System.ArgumentNullException)
               Actual:   typeof(System.NullReferenceException): Object reference not set to an instance of an object.
               Stack Trace:
                  E:\corefx\src\System.Reflection.TypeExtensions\tests\PropertyInfo\PropertyInfoGetAccessors1.cs(77,0): at System.Reflection.Tests.PropertyInfoGetAccessor1.<>c__DisplayClass5_0.<NegTest1>b__0()
         Finished:    System.Reflection.TypeExtensions.Tests
```

**Comment by pallavit** [comment]
@Priya91 Do you suspect any of the recent check-ins to cause this issue? I have not made any changes to this library's implementation and so just wanted to make sure,

**Comment by pallavit** [comment]
I looked at the issue and the behavior is expected and it is a test bug. I will create a PR once we get a green on submitting new items to the CI.

**Comment by pallavit** [comment]
Issue #6932 is fixed by #7003.
